progressive_rollout_demo/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: progressive_rollout_demo
+description: "Shows how to build a progressive patch rollout system with Shorebird tracks"
+publish_to: "none"
+
+version: 1.0.0+1
+
+environment:
+  sdk: ^3.5.4
+
+dependencies:
+  cloud_firestore: ^5.4.5
+  firebase_core: ^3.7.0
+  flutter:
+    sdk: flutter
+  package_info_plus: ^8.1.1
+  shared_preferences: ^2.3.3
+  shorebird_code_push:
+    path: /Users/bryanoltman/shorebirdtech/engine/src/third_party/updater/shorebird_code_push
```

Comment:
we'll need to use the published version before merging

Reply:
Yep. We'll also need to update the readme to point to the actual guide.